client/components/jetpack/jetpack-disconnected-wpcom/index.tsx

```diff
@@ -44,7 +44,7 @@ const JetpackDisconnectedWPCOM: FunctionComponent = () => {
 			<p>
 				{ preventWidows(
 					translate(
-						'Please visit {{siteUrl/}} to ensure your site loading correctly and reconnect Jetpack if necessary.',
+						'Please visit {{siteUrl/}} to ensure your site is loading correctly and reconnect Jetpack if necessary.',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.